### DISCUSSION TITLE
How do I remove a document from the AI's memory?

### DISCUSSION THREAD

**Discussion by lesleyklassen** [pashpashpash/vault-ai#22] [Q&A]
I am running my local version of Vault AI and find it very helpful.  I want to remove documents from memory.  I uploaded two text files, one with older information and one that was updated with new information and the AI can't distinguish the difference.  Your knowledge base is only as good as your data and my data needs cleaning. I'm pretty new to working with APIs and running local servers, which is what I am doing in this case.

**Comment by Retzilience** [comment]
I'm trying to figure out the easiest way myself. I'm not familiar with Pinecone but the index manager has several options on how to delete something from the index, it's just not clear to me how to request to delete the file I uploaded trough the local server.

If someone has more clarifying information about that it would really help.
